MAINTAINERS.md

```diff
@@ -1,13 +1,14 @@
 ## Maintainers
 
-| Maintainer | GitHub ID | Affiliation |
-| --------------- | --------- | ----------- |
-| Ashish Agrawal | [lezzago](https://github.com/lezzago) | Amazon |
-| Mohammad Qureshi | [qreshi](https://github.com/qreshi) | Amazon |
-| Sriram Kosuri | [skkosuri-amzn](https://github.com/skkosuri-amzn) | Amazon |
-| Bowen Lan | [bowenlan-amzn](https://github.com/bowenlan-amzn) | Amazon |
-| Rishabh Maurya | [rishabhmaurya](https://github.com/rishabhmaurya) | Amazon | 
-| Tianli Feng | [tlfeng](https://github.com/tlfeng) | Amazon |
-| Annie Lee | [leeyun-amzn](https://github.com/leeyun-amzn) | Amazon |
+| Maintainer       | GitHub ID                                         | Affiliation |
+|------------------|---------------------------------------------------| ----------- |
+| Ashish Agrawal   | [lezzago](https://github.com/lezzago)             | Amazon |
+| Mohammad Qureshi | [qreshi](https://github.com/qreshi)               | Amazon |
+| Sriram Kosuri    | [skkosuri-amzn](https://github.com/skkosuri-amzn) | Amazon |
```

Comment:
Please remove Sriram as he is no longer actively working on this

Reply:
I will consider cleanup as a separate exercises (and PR). We should come up with some process maintainers process this? Such as not active for 60 days? Thoughts?

Reply:
Created https://github.com/opensearch-project/common-utils/issues/141 to collect more thoughts on this.